QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+			"ceasn:hasGlossary": [
+				"res:acqfGlossary"
+			],
+			"asn:hasProgressionLevel": [
+				"res:acqfLevel01",
+				"res:acqfLevel02",
+				"etc"
+			]
+		},
+		{
+			"@id": "res:acqfLevel01",
+			"@type": "asn:ProgressionLevel",
+			"skos:prefLabel": { "en": "Level 1" },
+			"skos:definition": { "en": "The learning outcomes related to formal, non-formal, and informal learning at this level include:" }
+			"ceasn:facetedDescription": [
+				"res:knowledge_01",
+				"res:skill_01",
+				"res:autonomy_01"
+			]
+		},
+		{
+			"@id": "res:knowledge_01",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Simple knowledge, literacy, and numeracy" },
+			"???:facetType": [
+				"res:facetKnowledge"
+			],
+			"ceasn:isPartOf": [
+				"res:acqf"
+			]
+		},
+		{
+			"@id": "res:skill_01",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Simple communication, cognitive and practical skills required to follow simple instructions, and use simple, repetitive solutions to address simple problems" },
+			"???:facetType": [
+				"res:facetSkills"
+			],
+			"ceasn:isPartOf": [
+				"res:acqf"
+			]
+		},
+		{
+			"@id": "res:autonomy_01",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Highly structured, repetitive contexts under close supervision and guidance taking minimal responsibility for self" },
+			"???:facetType": [
+				"res:facetAutonomy"
+			],
+			"ceasn:isPartOf": [
+				"res:acqf"
+			]
+		},
+		{
+			"@id": "res:acqfLevel02",
+			"@type": "asn:ProgressionLevel",
+			"skos:prefLabel": { "en": "Level 2" },
+			"skos:definition": { "en": "The learning outcomes related to formal, non-formal and informal learning at this level include:" }
+			"???:facetedDescription": [
+				"res:knowledge_02",
+				"res:skill_02",
+				"res:autonomy_02"
+			]
+		},
+		{
+			"@id": "res:knowledge_02",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Basic knowledge incorporating comprehension and recall of factual and operational knowledge in some areas" },
+			"???:facetType": [
+				"res:facetKnowledge"
+			],
+			"ceasn:isPartOf": [
+				"res:acqf"
+			]
+		},
+		{
+			"@id": "res:skill_02",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Basic communication, cognitive and practical skills required to use concrete information, ideas and known solutions to address straight forward problems" },
+			"???:facetType": [
+				"res:facetSkills"
+			],
+			"ceasn:isPartOf": [
+				"res:acqf"
+			]
+		},
+		{
+			"@id": "res:autonomy_02",
+			"@type": "ceasn:Competency",
+			"ceasn:competencyText": { "en": "Structured contexts under limited supervision and guidance taking limited responsibility for self and group outcomes" },
+			"???:facetType": [
```

Comment:
Is `???` expected?

Reply:
Asume using the ceterms names space.  QFs are not part of ASN so it doesn't make sense to use the ASN name space.